server/daos/booking_dao.js

```diff
@@ -0,0 +1,70 @@
+// import database
+// import modules
+
+const db = require('../db/db.js');
+const Booking = require('../models/booking.js');
+
+const createBooking = function (row){
+    return new Booking(row.lecture_id,row.student_id,row.waiting,row.present,row.updated_at,row.deleted_at);
+}
+
+// it creates the booking table
+exports.createBookingTable = function() {
+    return new Promise ((resolve,reject) => {
+        const sql = `CREATE TABLE Bookings (lecture_id TEXT NOT NULL, student_id TEXT NOT NULL,
+                     waiting BOOLEAN NOT NULL DEFAULT (0) CHECK (waiting IN (0,1)),
+                     present BOOLEAN NOT NULL DEFAULT (0) CHECK (present IN (0,1)),
+                     updated_at TEXT,deleted_at TEXT, PRIMARY KEY(lecture_id,student_id),
```

Comment:
Should it be timestamp type? Or it is not available in sqlite3?

Reply:
Yes, I remember that in the office project we had something like this initial_time `TIMESTAMP NOT NULL DEFAULT (datetime('now','localtime'))`

Reply:
no I don't think so because I looked at sqlite documentation and it says that timestamp and varchar don't exist in sqlite and they are approximated with text, infact I'm a bit confused

Reply:
@LucaAntoniotti you are right, just checked